qtop.py

```diff
@@ -1844,6 +1868,11 @@ def _sort_worker_nodes(self):
 
         return self.worker_nodes
 
+    def _calculate_jobs_per_node(self, workernode_dict):
+        for node in workernode_dict:
+            node_job_set = set(workernode_dict[node]['core_job_map'].values())
+            workernode_dict[node]['node_job_set'] = node_job_set
+
```

Comment:
all good, just remain with the question if such additions could ever be generalized in configuration files
(it could well be, they can't)